src/common/Error/styled.js

```diff
@@ -0,0 +1,64 @@
+import styled from "styled-components";
+import { SectionTitle } from '../Section/Section';
+import { ReactComponent as ErrorSVG } from "../../assets/icon-error.svg";
+
+export const Container = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
+    padding-top: 180px;
+    
+    @media (max-width: 767px) {
+        padding-top: 54px;
+        gap: 16px;
+	}
+`;
+
+export const StyledErrorIcon = styled(ErrorSVG)`
+    width: 120px;
+	height: 120px;
+    pointer-events: none;
+
+    @media (max-width: 767px) {
+        width: 60px;
+	    height: 60px;
+	}
+`;
+
+export const Header = styled(SectionTitle)`
+    margin: 0;
```

Comment:
i would add `text-align: center;` to Header - on smallest devices title is to long and aliging ites to center via flex stops working properly 🙂

Reply:
Absolutely, great catch! 🕵️‍♀️ I've already made the adjustment 🙂